tests/utils.go

```diff
@@ -4564,33 +4564,37 @@ func waitForConfigToBePropagated(resourceVersion string) {
 func WaitForConfigToBePropagatedToComponent(podLabel string, resourceVersion string, compareResourceVersions compare) {
 	virtClient, err := kubecli.GetKubevirtClient()
 	PanicOnError(err)
+	errAdditionalInfo := fmt.Sprintf("component: \"%s\"", strings.TrimPrefix(podLabel, "kubevirt.io="))
 
-	EventuallyWithOffset(3, func() bool {
+	EventuallyWithOffset(3, func() error {
 		pods, err := virtClient.CoreV1().Pods(flags.KubeVirtInstallNamespace).List(context.Background(), metav1.ListOptions{LabelSelector: podLabel})
+
 		if err != nil {
-			log.DefaultLogger().Reason(err).Infof("Failed to fetch pods.")
-			return false
+			return fmt.Errorf("failed to fetch pods. %s", errAdditionalInfo)
 		}
 		for _, pod := range pods.Items {
+			errAdditionalInfo += fmt.Sprintf(", pod: \"%s\"", pod.Name)
```

Comment:
if I read the code correct this will add every pod name to errAdditionalInfo accumulating them.

Reply:
Thanks!
PR: https://github.com/kubevirt/kubevirt/pull/5776